backend/elasticsearch/nb-configuration.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't include IDE configuration in your commits. You can add this line to the `.gitignore` file to have these files automatically ignored:

```
nb-configuration.xml
```

Reply:
That would require a global patch in .gitignore.

`# This patch file was generated by NetBeans IDE
# It uses platform neutral UTF-8 encoding and \n newlines.
--- a/.gitignore
+++ b/.gitignore
@@ -45,3 +45,7 @@
 # Maven wrapper files
 .mvn
 mvnw*
+/backend/elasticsearch/nb-configuration.xml
+/backend/lucene/nb-configuration.xml
+/engine/nb-configuration.xml
+/nb-configuration.xml
\ No newline at end of file`

Reply:
Alright, I added it to `.gitignore`. Please remove the files from your commit and rebase on master: they should be ignored by git from now on.